src/g3log/logmessage.hpp

```diff
@@ -87,7 +88,7 @@ namespace g3 {
       // Complete access to the raw data in case the helper functions above
       // are not enough.
       //
-      std::time_t _timestamp;
+      Timestamp _timestamp;
```

Comment:
That's right. Custom Timestamp is gone in the my last commit:
`timespec _timestamp;`